app/web/markdown/blog/2024/05/20/introducing-strong-verification.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Introducing Strong Verification
+slug: introducing-strong-verification
+description: We are excited to announce a new verification feature called "Strong Verification". This feature increases Trust and Safety on the platform by allowing users to verify their information using a biometric passport.
+date: 2024/05/20
+author: Aapeli
+# todo: picture of biometric passport
+share_image: https://couchers.org/img/share.jpg
+---
+
+Today we're announcing a new feature called Strong Verification. You can complete Strong Verification today using your biometric passport and an NFC enabled mobile phone.
+
+Trust and Safety are a core tenet of usability for any couch surfing platform. At Couchers, we've spent a lot of time thinking about how we could make the platform safer and give people more control in order to use the platform according to their needs and preferences. Today we're announcing a new feature that gives users more power and peace of mind when using Couchers.org.
```

Comment:
Today we're announcing a new feature that improves verification and gives users peace of mind when using Couchers.org.